EMDS: Write to flash bypassing the flash driver (#9164)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    * Ability to use buttons to send get and set messages for a sensor setting, as well as a get message for a sensor descriptor.

* :ref:`bluetooth_mesh_light_lc`

>   * Removed :c:func:`bt_disable` call from the interrupt context before calling :c:func:`emds_store`.
>   * :c:func:`mpsl_uninit` has been replaced with :c:func:`mpsl_lib_uninit`.

Reviewer: ```suggestion
  * Removed the :c:func:`bt_disable` function call from the interrupt context before calling the :c:func:`emds_store` function.
  * Replaced the :c:func:`mpsl_uninit` function with :c:func:`mpsl_lib_uninit`.
```

* Added:
  * :ref:`lib_sfloat` library.

* :ref:`emds_readme`:

>   * Removed the internal thread for storing emergency data.
>     The emergency data is now stored in :c:func:`emds_store`.
>   * Changed the library implementation to bypass the flash driver when storing emergency data.
>     This allows to call :c:func:`emds_store` from an interrupt context.

Reviewer: ```suggestion
  * Removed the internal thread for storing the emergency data.
    The emergency data is now stored by the :c:func:`emds_store` function.
  * Changed the library implementation to bypass the flash driver when storing the emergency data.
    This allows calling the :c:func:`emds_store` function from an interrupt context.
```
